core/src/main/c/share/ooo_dispatch.h

```diff
@@ -26,6 +26,20 @@
 #define QUESTDB_OOO_DISPATCH_H
 
 #include "dispatcher.h"
+#include "util.h"
+
+typedef struct index_l {
+    int64_t ts;
+    uint64_t i;
+
+    uint64_t operator>>(uint64_t shr) const {
```

Comment:
sign extension in right shift, is it expected? same for `operator&`

Reply:
I removed the operators; they are not really necessary